package.json

```diff
@@ -20,7 +20,9 @@
     "abortcontroller-polyfill": "^1.4.0",
     "core-js": "^3.5.0",
     "cross-fetch": "^3.0.4",
-    "debug": "^4.1.1"
+    "crypto-js": "^3.1.9-1",
+    "debug": "^4.1.1",
+    "sha.js": "^2.4.11"
```

Comment:
Yeah, we endup using `sha.js` to do the `sha256`, since the `window.crypto.subtle.digest` is not available for all browser this client support (ie10, ie11).

Reply:
Isn't the sha256 encoding done by crypto-js/sha256?

Reply:
Ah, yeah, right